.github/workflows/agent-review.yml

```
@@ -0,0 +1,165 @@
+name: Agent Review
+
+on:
+  pull_request:
+    branches: [tauri-base]
+    types: [opened, synchronize, ready_for_review, reopened]
+  pull_request_review:
+    branches: [tauri-base]
```

actionlint (1.7.12)

[error] "branches" filter is not available for pull_request_review event. it is only for merge_group, push, pull_request, pull_request_target, workflow_run events

(events)